RhysMP.md

```diff
@@ -0,0 +1,37 @@
+---
+name: "@RhysMP"
+project: "DonkeyCar RC car"
+---
+
+# DonkeyCar RC car
+
+## Summary
+
+I plan on building a RC car which can  drive itself and also be controlled from mobile. I'm super excited as I have always wanted to experiemnt with a rasberry pi and see what amazing things I could do with it. I also have always thought RC cars were fasinating. 
+
+## Plan
+
+First I will order all of the products. 
+
+Then I will send all the CAD files required for this car to my HS's Physics teacher to 3D print using the makerbots we have on campus.
+
+I will then clean up all of these prints.
+
+I will attach the parts together. (Pi > 3d printed plate) (Camera to Adatper)
+
+Then I will install the Donkeycare software and configure it all.
```

Comment:
https://github.com/hackclub/winter/pull/118#discussion_r1059120198

Could you update your proposal to reflect this (writing custom firmware)?

Reply:
Done, @exu3